bin/hit/test/env

```diff
@@ -14,6 +14,12 @@ select_test_env() {
   done
   prompt=${prompt%,}  # Remove the trailing comma
 
+  # Return early if the environement is specified as argument
+  if [[ " ${env_array[*]} " =~ [[:space:]]${1}[[:space:]] ]]; then
```

Comment:
why do you need the spaces?

Reply:
to make sure we not only match parts of the wagon name, but only the full name (since it needs the full name).

So `"wagon_1 core" =~ cor` would match, but with the spaces it won't.

Reply:
interesting, thanks!